install/sles-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Installing Kata Containers on SLES
+
+Note: Kata Containers is available for SLES version 12 SP3.
+
+This step is only required in case Docker is not installed on the system.
+1. Install the latest version of Docker with the following commands:
+
+```bash
+$ sudo zypper -n install libcgroup1
+$ curl -OkL https://yum.dockerproject.org/repo/main/opensuse/13.2/Packages/docker-engine-17.05.0.ce-1.x86_64.rpm
+$ sudo rpm -ivh docker-engine-17.05.0.ce-1.x86_64.rpm
+```
+
+2. Install the Kata Containers components with the following commands:
+
+  **Note:** The repository redirects the download content to use `http`, be aware that this installation channel is not secure.
+```bash
+$ sudo -E zypper addrepo \
+http://download.opensuse.org/repositories/home:/katacontainers:/release/SLE_12_SP3/home:katacontainers:release.repo
+$ sudo -E zypper -n --gpg-auto-import-keys install kata-runtime kata-proxy kata-shim
+```
+
+3. Configure Docker to use Kata Containers by default with the following commands:
+
+```bash
+$ sudo mkdir -p /etc/systemd/system/docker.service.d/
+$ cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
```

Comment:
add ` --containerd /run/containerd/containerd.sock` or change to use `/etc/sysconfig/docker` instead of the docker service config file.